src/components/members.component.js

```diff
@@ -78,28 +121,20 @@ const MembersListComponent = ({
   navigation,
   authUser,
 }: Props) => (
-  <View style={[styles.container, containerStyle && containerStyle]}>
-    <Text style={smallTitle ? styles.sectionTitleSmall : styles.sectionTitle}>
-      {title}
-    </Text>
+  <Container style={containerStyle}>
```

Comment:
Scanned the code to see where `containerStyle` is passed, it's only used in `<IssueDescription>` as follows:

```
<MembersList
  title={translate('issue.main.assignees', locale)}
  members={issue.assignees}
  containerStyle={{ marginTop: 0, paddingTop: 0, paddingLeft: 0 }}
  smallTitle
  navigation={navigation}
/>
```

Could you introduce a new boolean prop, like "noMargin" and get rid of `containerStyle`?

The `Container` styled component should be updated to read this new props and add the extra css declarations instead. IssueDescription will need to be updated and be part of this PR of course.

Let me know if this is unclear to you, I'll be happy to help